src/qtfirebaseadmob.cpp

```diff
@@ -1065,7 +1073,8 @@ void QtFirebaseAdMobInterstitial::show()
  */
 
 QtFirebaseAdMobRewardedVideoAd::QtFirebaseAdMobRewardedVideoAd(QObject* parent):
-    QtFirebaseAdMobBase(parent)
+    QtFirebaseAdMobBase(parent),
+    firebase::admob::rewarded_video::Listener()
```

Comment:
On all my test devices, with the current QtFirebase master branch, I get this (in regards to the "missing OnRewarded" call you mention):
```
qtfirebaseadmob.cpp:1220 (virtual void QtFirebaseAdMobRewardedVideoAd::OnRewarded(firebase::admob::rewarded_video::RewardItem)): QtFirebaseAdMobRewardedVideoAd(0x6509c418) "Rewarding user of coins with amount 10"
```
So, yes - the callback is fired